generator/testdata/google/example/library/v1/library_service_stub.gapic.cc.baseline

```diff
@@ -60,3 +62,79 @@ LibraryServiceStub::GetBigBook(
 
 LibraryServiceStub::~LibraryServiceStub() {}
 
+namespace {
+class DefaultLibraryServiceStub : public LibraryServiceStub {
+ public:
+    DefaultLibraryServiceStub(std::unique_ptr<LibraryService::StubInterface> grpc_stub)
```

Comment:
I should have seen this previously, but I think this should be just 2 space indented, not 4. So everything is 2 space indent, except the {public, protected, private} specifiers in a class, which are 1 space. Like e.g. https://github.com/googleapis/google-cloud-cpp/blob/master/google/cloud/bigtable/instance_admin.h#L43

Feel free to save this for a future PR if you prefer.

Reply:
I'll bump it to a standalone whitespace fix immediately after this one lands.